src/ray/protobuf/ray_client.proto

```diff
@@ -367,6 +367,9 @@ message DataRequest {
     PrepRuntimeEnvRequest prep_runtime_env = 7;
     ConnectionCleanupRequest connection_cleanup = 8;
     AcknowledgeRequest acknowledge = 9;
+    ClientTask task = 10;
+    TerminateRequest terminate = 11;
+    ClientListNamedActorsRequest list_named_actors = 12;
```

Comment:
It looks like we move some functions to the data client. Should we clean up the raylet one, like Schedule/ListNamedActors ?

Reply:
It is a bit confusing but the handlers of Schedule / ListNamedActors in server.py are called by the streaming Datapath handler to handle the corresponding logic e.g. ClientTask / ClientTaskTicket. Before this change, Get / Put etc are in the same situation, where client - server interacts via Datapath but Get / Put handlers are still exposed. This can be cleaned up in a future PR, after we have a more stable client / server protocol.

Reply:
Got it. It makes sense!